### PR TITLE
New Package: libax25-0.0.12rc4

### PR DESCRIPTION
**Pull request by the-maldridge**
Part of the AX.25 Amatuer Radio packet routing stack.

### CODE REVIEW

File: srcpkgs/libax25/template [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+build_style=gnu-configure
+short_desc="A set of functions making it easier to write hamradio programs"
+maintainer="Michael Aldridge <aldridge.mac@gmail.com>"
+license="LGPL-2"
+homepage="http://www.linux-ax25.org/wiki/Libax25"
+distfiles="http://www.linux-ax25.org/pub/libax25/libax25-0.0.12-rc4.tar.gz"
+checksum=051bd736c3f3f3b242b3efea91af37f2e8d5afaebb5fe6e1df050bb1f6aeb986
+
+case "$XBPS_TARGET_MACHINE" in
+	*-musl)	makedepends+=" musl-bootstrap"
```
Comment (dominikh) [inline srcpkgs/libax25/template]: No.

File: srcpkgs/libax25/template [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+_libax25_rc=rc4
+version="${_libax25_ver}${_libax25_rc}"
+revision=1
+wrksrc="${pkgname}-${_libax25_ver}-${_libax25_rc}"
+build_style=gnu-configure
+short_desc="A set of functions making it easier to write hamradio programs"
+maintainer="Michael Aldridge <aldridge.mac@gmail.com>"
+license="LGPL-2"
+homepage="http://www.linux-ax25.org/wiki/Libax25"
+distfiles="http://www.linux-ax25.org/pub/libax25/libax25-0.0.12-rc4.tar.gz"
```
Comment (dominikh) [inline srcpkgs/libax25/template]: Use `${version}` in the URL.

File: srcpkgs/libax25/patches/daemon.patch [hunk 3]
```diff
@@ -0,0 +1,10 @@
+--- daemon.bak	2016-01-16 20:34:52.682900674 -0600
```
Comment (dominikh) [inline srcpkgs/libax25/patches/daemon.patch]: Call this file `musl.patch` so we know what it's for.

File: srcpkgs/libax25/template [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+		;;
+esac
+
+libax25-devel_package() {
+	short_desc+=" - development files"
+	depends="libax25>=${version}_${revision}"
+	pkg_install() {
+		vmove usr/include
+		vmove usr/lib/*.a
+		vmove usr/share/man/man3
```
Comment (dominikh) [inline srcpkgs/libax25/template]: `*.so` (unversioned .so) also need to go in the -devel package.

**Comment by the-maldridge** [comment]
Updated with the requested changes.

**Comment by xtraeme** [comment]
```
checking for zlib.h... no
```
Try adding `makedepends="zlib-devel"`

**Comment by the-maldridge** [comment]
I have added the requested library.

**Pull request closed without merging**